README.md

```diff
@@ -11,29 +11,36 @@ These components are significantly more complex than the ones found in the [MUI
 They feature advanced functionality for data-rich applications and a wide range of other use cases.
 
 MUI X is **open core**—base components are MIT-licensed, while more advanced features require a Pro or Premium commercial license.
-See the [Licensing page](https://mui.com/x/introduction/licensing/) for details.
+See [Licensing](https://mui.com/x/introduction/licensing/) for details.
 
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6293/badge)](https://bestpractices.coreinfrastructure.org/projects/6293)
 
 ## Components
 
-- [Data Grid](https://mui.com/x/react-data-grid/)
-- [Date and Time Pickers](https://mui.com/x/react-date-pickers/getting-started/)
-- [Charts](https://mui.com/x/react-charts/)
-
-## Installation
+- [Data Grid](#data-grid)
+- [Date and Time Pickers](#date-and-time-pickers)
+- [Charts](#charts)
 
 ### Data Grid
 
-Read the Data Grid [Installation instructions](https://mui.com/x/react-data-grid/getting-started/#installation) in the MUI X docs.
+- [Documentation](https://mui.com/x/react-data-grid/)
+- [Installation instructions](https://mui.com/x/react-data-grid/getting-started/#installation)
+
+![Data Grid)](https://github.com/mui/mui-x/assets/3309670/45a1c935-dda1-48e7-bfaf-ef85c4ca9611)
```

Comment:
I wonder about the illustrations. It's a great way to quickly get introduced to what each component is about, but on the other hand, it will get out of date, relatively quickly. So great for the DX but maybe an opportunity cost to maintain fresh.

If we had them, I think that the must be at a x2 pixel CSS ratio, these are at a x1 pixel ratio right now.

Reply:
> If we had them, I think that the must be at a x2 pixel CSS ratio, these are at a x1 pixel ratio right now.

I'll let someone with a x2 pixell CSS ratio screen do it then

Reply:
Talking about outdated visuals, the one on [the store](https://mui.com/store/items/mui-x-pro/) are not great either.
Maybe we could settle on a shared set of 2-3 visuals (does not have to be GIFs) and re-use them everywhere.

That way it would be simpler to update them when needs be.

Reply:
I have tried something. It's a challenge when we are limited to gif. Maybe static images would be better.